Cargo.toml

```diff
@@ -56,6 +59,7 @@ headers = ["headers-shared", "headers-um", "headers-vc"]
 "um-combaseapi" = ["shared-basetsd", "shared-minwindef", "shared-wtypesbase", "um-objidlbase", "ole32"]
 "um-consoleapi" = ["shared-minwindef", "um-wincon", "um-winnt", "kernel32"]
 "um-minwinbase" = ["shared-ntstatus"]
+"um-ncrypt" = ["shared-basetsd", "shared-sspi"]
```

Comment:
This should also depend on `"ncrypt"` because it has functions from that library.